habitat_baselines/il/trainers/eqa_trainer.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from habitat import logger
+from habitat_baselines.common.base_trainer import BaseILTrainer
+from habitat_baselines.common.baseline_registry import baseline_registry
+from habitat_baselines.common.tensorboard_utils import TensorboardWriter
+
+
+@baseline_registry.register_trainer(name="eqa")
+class EQATrainer(BaseILTrainer):
```

Comment:
Is this used anywhere?

Reply:
Not yet. I had made a template for the future where I'd put the joint trainer.
I can remove it if you like.